firefox_desktop/explores/newtab_interactions.explore.lkml

```diff
@@ -17,4 +17,182 @@ explore: newtab_interactions {
+        newtab_interactions.search_engine,
+        countries.tier,
+      ]
+      measures: [
+        newtab_interactions.visits_with_search,
+        newtab_interactions.visits_with_tagged_search_ad_impression,
+        newtab_interactions.visits_with_tagged_search_ad_click,
+        newtab_interactions.visits_with_tagged_follow_on_search_ad_impression,
+        newtab_interactions.visits_with_tagged_follow_on_search_ad_click,
+        newtab_interactions.visits_with_any_ad_click,
+        newtab_interactions.visits_with_any_ad_impression,
+
+        newtab_interactions.sum_searches,
+        newtab_interactions.sum_tagged_search_ad_impressions,
+        newtab_interactions.sum_tagged_search_ad_clicks,
+        newtab_interactions.sum_tagged_follow_on_search_ad_impressions,
+        newtab_interactions.sum_tagged_follow_on_search_ad_clicks,
+        newtab_interactions.sum_all_search_ad_clicks,
+        newtab_interactions.sum_all_search_ad_impressions,
+
+        newtab_interactions.clients_with_search,
+        newtab_interactions.clients_with_tagged_search_ad_impression,
+        newtab_interactions.clients_with_tagged_search_ad_click,
+        newtab_interactions.clients_with_tagged_follow_on_search_ad_impression,
+        newtab_interactions.clients_with_tagged_follow_on_search_ad_click,
+        newtab_interactions.clients_with_any_ad_click,
+        newtab_interactions.clients_with_any_ad_impression,
+      ]
+      filters: [newtab_interactions.submission_date: "after 2022-07-01"]
+    }
+
+    materialization: {
+      sql_trigger_value:
+      SELECT
+        MAX(last_modified_time)
+      FROM
+        moz-fx-data-shared-prod.telemetry_derived.INFORMATION_SCHEMA.PARTITIONS
+      WHERE
+        table_name = "newtab_interactions_v1";;
+      increment_key: newtab_interactions.submission_date
+      increment_offset: 1
+    }
+  }
+
+  aggregate_table: rollup__newtab_interactions_topsites_per_day {
+    query: {
+      dimensions: [
+        newtab_interactions.submission_date,
+        newtab_interactions.channel,
+        newtab_interactions.country_code,
+        countries.tier,
+      ]
+      measures: [
+        newtab_interactions.visits_with_topsite_click,
+        newtab_interactions.visits_with_topsite_impression,
+        newtab_interactions.visits_with_sponsored_topsite_click,
+        newtab_interactions.visits_with_sponsored_topsite_impression,
+        newtab_interactions.sum_topsite_clicks,
+        newtab_interactions.sum_topsite_impressions,
+        newtab_interactions.sum_sponsored_topsite_clicks,
+        newtab_interactions.sum_sponsored_topsite_impressions,
+        newtab_interactions.clients_with_topsite_click,
+        newtab_interactions.clients_with_topsite_impression,
+        newtab_interactions.clients_with_sponsored_topsite_click,
+        newtab_interactions.clients_with_sponsored_topsite_impression,
+      ]
+      filters: [newtab_interactions.submission_date: "after 2022-07-01"]
```

Comment:
Can we also filter this to only countries with Topsites enabled?

Reply:
We don't currently have a list of those iirc